dascore/utils/models.py

```
@@ -1,305 +1,16 @@
-        Unset (NaT) starts are unbounded past; unset ends are ongoing.
-        """
-        s1, e1, s2, e2 = (
-            self.start_time,
-            self.end_time,
-            other.start_time,
-            other.end_time,
-        )
-        first_starts_before = pd.isnull(e2) or pd.isnull(s1) or s1 < e2
-        second_starts_before = pd.isnull(e1) or pd.isnull(s2) or s2 < e1
-        return bool(first_starts_before and second_starts_before)
+__all__ = [*_models_all, "BaseModel"]
```

Ruff (0.16.1)

[error] Invalid object in `__all__`, must contain only strings

(PLE0604)